### ISSUE TITLE
[Suggestion]: Toggleable Radius Particle Effect

### ISSUE THREAD

**Issue by SSgamingYT** [nonamecrackers2/crackers-wither-storm-mod#2122]
### Guidelines

- [X] By checking this box, I am saying that I have read the guidelines and believe to be following them to the best of my ability

### Description of feature request

A particle effect that is on the ground, it marks the radius that the wither storm is currently able to grab block clusters. The particle in question is the same as the tractor beam particle, just less frequent and more spread out

**Comment by Kyleo3345** [comment]
Nice. It would tell you to watch out for them and not get caught more easily.

**Comment by SSgamingYT** [comment]
> Nice. It would tell you to watch out for them and not get caught more easily.

Not really, it's a particle effect that's on the ground around the wither storm, not from the beams. The beams already have a particle effect

**Comment by nonamecrackers2** [comment]
This would have to be toggleable not something that would be visible playing through the mod normally
